src/main/scala/com/datastax/sparkstress/DataGenerator.scala

```diff
@@ -40,7 +40,7 @@ object RowGenerator {
       val ckeysPerPkey = numTotalOps / numTotalKeys
 
       for ( pk <- (0L until keysPerPartition); ck <- (0L until ckeysPerPkey)) yield
-        new ShortRowClass((start + pk), (ck).toString, r.nextString(20), r.nextString(20))
+        new WideRowClass((start + pk), (ck).toString, r.nextString(20), r.nextString(20), r.nextString(20), r.nextString(20), r.nextString(20), r.nextString(20), r.nextString(20), r.nextString(20))
```

Comment:
I know i had the naming on this strange before but I kinda liked have it be wide but only in the sense of "Partition wide" But i'm alright with this row format as well